src/views/webview-app/components/resources-panel/resources-panel.less

```
@@ -0,0 +1,124 @@
+
+.resources-panel {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 10;
+}
+
+.resources-panel-background {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: -1;
+  background-color: rgba(0, 0, 0, 0.5);
+  animation-duration: 250ms;
+  animation-direction: forwards;
+  animation-name: backgroundDarkenAnimation;
+}
+
+@keyframes backgroundDarkenAnimation {
+  from {
+    width: rgba(0, 0, 0, 0.1);
```

```suggestion
    background-color: rgba(0, 0, 0, 0.1);
```
Seems like a copy and paste error? 😅